tests/flow/tests_withGears.py

```diff
@@ -0,0 +1,311 @@
+import redis
+
+from includes import *
+import os
+from functools import wraps
+
+
+def skip_if_gears_not_loaded(f):
+    @wraps(f)
+    def wrapper(env, *args, **kwargs):
+        con = env.getConnection()
+        modules = con.execute_command("MODULE", "LIST")
+        if b'rg' in [module[1] for module in modules]:
+            return f(env, *args, **kwargs)
+        try:
+            platform = paella.Platform()
+            redisgears_dir = "{ROOT}/bin/{PLATFORM}/RedisGears".format(ROOT=ROOT, PLATFORM=platform.triplet())
+            redisgears_path = os.path.join(redisgears_dir, 'redisgears.so')
+            python_plugin_path = os.path.join(redisgears_dir, 'plugin/gears_python.so')
+            ret = con.execute_command('MODULE', 'LOAD', redisgears_path, 'Plugin', python_plugin_path, 'CreateVenv',
```

Comment:
It would be better to wrap the _try_ around the execute_command line, since that's what can fail. 

Defensively, I might check for the redisgears_path and python_plugin_path (if os.path.isdir(<thing>) prior to executing the command. Then we can exit with meaningful text about missing directories.